zephyr/CMakeLists.txt

```diff
@@ -292,8 +292,8 @@ if (CONFIG_SOC_SERIES_INTEL_ADSP_ACE)
 		set(PLATFORM "meteorlake")
 	elseif(CONFIG_SOC_INTEL_ACE20_LNL)
 		set(PLATFORM "lunarlake")
-	elseif(CONFIG_SOC_INTEL_ACE30_PTL)
-		set(PLATFORM "pantherlake")
```

Comment:
No need to change, but we do have prior examples of using the first product/variant name to name new platforms. E.g. all Intel "cAVS2.5" are referred to as "tigerlake" (e..g src/platform/tigerlake in SOF). This is not fully correct, but there is a system to it, using the name of the first submitted veriant. 

In retrospect, this could have been used here as well and avoid the ace30_ptl->ace30 rename effort, but alas, that's now done, so let's just complete this.
FYI @nashif @abonislawski @marcinszkudlinski

Reply:
@kv2019i, I'm going to push initial PR for NVL. What is preferred: to continue with new style and have platform named "ace40" or go the old way and name it "novalake"?